features/networking/ovn.feature

```diff
@@ -465,3 +465,61 @@ Feature: OVN related networking scenarios
     Then the step should succeed
     # the iface-id should be quoted if it starts with a digit
     And the output should contain "<%= project.name %>"
+
+
+# @author huirwang@redhat.com
+# @case_id OCP-38132
+@admin
+@destructive
+Scenario: Should no intermittent packet drop from pod to pod after change hostname
+  Given I store the schedulable workers in the :nodes clipboard
+  Given admin uses the "openshift-ovn-kubernetes" project
+  And I store the hostname from external ids in the :original_hostname clipboard on the "<%= cb.nodes[0].name %>" node
+  Then the step should succeed
+  And I store the "short" hostname in the :short_hostname clipboard for the "<%= cb.nodes[0].name %>" node
+  Then the step should succeed
+
+  # Configure short hostname to the external_ids for the ovn node
+  When I set "<%= cb.short_hostname %>" hostname to external ids on the "<%= cb.nodes[0].name %>" node
+  Then the step should succeed
+  Given I register clean-up steps:
+  """
+  When I set "<%= cb.original_hostname%>" hostname to external ids on the "<%= cb.nodes[0].name %>" node
+  Then the step should succeed
+  """
+
+  # Check short hostname was set
+  Given I store the ovnkube-master "north" leader pod in the clipboard
+  Given I wait up to 15 seconds for the steps to pass:
+  """
+  And admin executes on the pod "northd" container:
+    | bash | -c | ovn-sbctl --no-leader-only list chassis \|grep "<%= cb.short_hostname %>" |
```

Comment:
if we want exact we could use `find`   This is more what the OVN code itself uses.

`| ovn-sbctl | --no-leader-only | --format=csv | --data=bare  |  --no-heading  |  --columns=hostname | find | chassis | hostname=<%= cb.short_hostname %> |`